app/src/main/java/com/example/tapp/Record.java

```diff
@@ -27,19 +29,75 @@ private void handleTimerComplete() {
         switch (this.timersComplete) {
             case 1:
                 // Clear screen and set screen to waiting period.
-                timerLabel.setText("Please switch to your right hand.");
+                timerLabel.setText("Please repeat left hand again.");
                 countText.setText("");
                 this.waitingTimer.start();
+                lTotal += lCount;
+                lCount = 0;
                 break;
             case 2:
                 // End the waiting period and allow the next tap to start the timer.
-                timerLabel.setText("Tap the screen with your right hand.\nYou have 10 seconds to tap as quickly as possible.");
+                timerLabel.setText("Tap the screen with your left hand.\nYou have 10 seconds to tap as quickly as possible.");
                 this.countdownReadyToStart = true;
                 break;
             case 3:
+                // Clear screen and set screen to waiting period.
```

Comment:
Are all these case statements necessary? I'd imagine it'd be easier to reuse the case statements for each trial.